test/integration/runonce_duration_test.go

```diff
@@ -0,0 +1,101 @@
+// +build integration
+
+package integration
```

Comment:
That does assume that the test executor isn't running the entire file/package, correct? That is, you assume the tests will be run method-per-process by the `test-integration.sh` script?

That's fine, just trying to understand. It's unfortunate but probably necessary.